pkg/oc/cli/importimage/importimage.go

```diff
@@ -481,23 +526,33 @@ func (o *ImportImageOptions) newImageStreamImportAll(stream *imageapi.ImageStrea
 	return isi
 }
 
-func (o *ImportImageOptions) newImageStreamImportTags(stream *imageapi.ImageStream, tags map[string]string) *imageapi.ImageStreamImport {
+func (o *ImportImageOptions) newImageStreamImportTags(stream *imageapiv1.ImageStream, tags map[string]string) *imageapiv1.ImageStreamImport {
 	isi, streamInsecure := o.newImageStreamImport(stream)
 	for tag, from := range tags {
 		insecure := streamInsecure
 		scheduled := o.Scheduled
-		oldTag, ok := stream.Spec.Tags[tag]
-		if ok {
+
+		oldTagFound := false
+		var oldTag imageapiv1.TagReference
+		for _, t := range stream.Spec.Tags {
+			if t.Name == tag {
```

Comment:
@mfojtik is there a helper I can use here to match a TagReference? I'm pretty sure matching on `t.Name` is not correct